website/source/docs/configuration/providers.html.md

```diff
@@ -117,3 +117,14 @@ KEY {
 	CONFIG
 }
 ```
+
+## Interpolation
+Providers support [interpolation syntax](/docs/configuration/interpolation.html) allowing dynamic configuration at run time.
+
+```
+provider "aws" {
+	region = "${var.aws_region}"
```

Comment:
Good catch, I need to update my editor settings, I normally always use 2 spaces in place of a tab and not sure why this did not get picked up for the markdown.